client/landing/stepper/declarative-flow/domain-upsell.ts

```diff
@@ -46,6 +46,12 @@ const domainUpsell: Flow = {
 			window.location.assign( location );
 		};
 
+		function goBack() {
```

Comment:
@alshakero noted _"Unless someone starts passing parameters to goBack which would really throw a wrench."_:

Could this be added as a comment to guide future developers?

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/77168